buildout-mercator.cfg

```diff
@@ -10,6 +10,11 @@ frontend.static_dir = src/mercator/mercator/build
 frontend_package_name = mercator
 backend_package_name = adhocracy_mercator
 
+[thentos_captcha]
+required = true
+backend_url = http://localhost:6542/
+frontend_url = http://localhost:6542/
+
```

Comment:
There is no need to make these values configurable with buildout, the default values are good enough for development, or?